*** ./smali/com/android/internal/widget/ILockSettings.smali	2015-07-25 14:08:57.590876370 +0530
--- /home/jaskaran/Android/Patchrom/nexus4/temp/new_smali/framework2.jar.out/./smali/com/android/internal/widget/ILockSettings.smali	2015-07-25 14:05:53.994215697 +0530
***************
*** 111,116 ****
--- 111,124 ----
      .end annotation
  .end method
  
+ .method public abstract setRawLockPassword([BI)V
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract setString(Ljava/lang/String;Ljava/lang/String;I)V
      .annotation system Ldalvik/annotation/Throws;
          value = {
