*** ./smali/com/android/internal/widget/ILockSettings.smali	2015-10-17 10:50:44.792757368 -0400
--- /home/jaskaran/Android/Patchrom6/nexus4/temp/new_smali/framework2.jar.out/./smali/com/android/internal/widget/ILockSettings.smali	2015-10-17 10:45:48.132770183 -0400
***************
*** 111,116 ****
--- 111,124 ----
      .end annotation
  .end method
  
+ .method public abstract setRawLockPassword([BI)V
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract setString(Ljava/lang/String;Ljava/lang/String;I)V
      .annotation system Ldalvik/annotation/Throws;
          value = {
