*** ./smali/android/media/MediaMetadataEditor.smali	2015-10-17 10:51:31.636088677 -0400
--- /home/jaskaran/Android/Patchrom6/nexus4/temp/new_smali/framework.jar.out/./smali/android/media/MediaMetadataEditor.smali	2015-10-17 10:46:29.702768387 -0400
***************
*** 140,145 ****
--- 140,151 ----
  
      sget-object v0, Landroid/media/MediaMetadataEditor;->METADATA_KEYS_TYPE:Landroid/util/SparseIntArray;
  
+     const/16 v1, 0x3e8
+ 
+     invoke-virtual {v0, v1, v2}, Landroid/util/SparseIntArray;->put(II)V
+ 
+     sget-object v0, Landroid/media/MediaMetadataEditor;->METADATA_KEYS_TYPE:Landroid/util/SparseIntArray;
+ 
      const/16 v1, 0x64
  
      invoke-virtual {v0, v1, v5}, Landroid/util/SparseIntArray;->put(II)V
