*** ./smali/com/android/internal/app/IAppOpsService.smali	2015-10-17 10:50:37.896090999 -0400
--- /home/jaskaran/Android/Patchrom6/nexus4/temp/new_smali/framework2.jar.out/./smali/com/android/internal/app/IAppOpsService.smali	2015-10-17 10:45:42.152770441 -0400
***************
*** 23,28 ****
--- 23,36 ----
      .end annotation
  .end method
  
+ .method public abstract checkOperationInternal(IILjava/lang/String;)I
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract checkPackage(ILjava/lang/String;)I
      .annotation system Ldalvik/annotation/Throws;
          value = {
***************
*** 89,94 ****
--- 97,110 ----
      .annotation system Ldalvik/annotation/Throws;
          value = {
              Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
+ .method public abstract registerCallback(Lcom/android/internal/app/IOpsCallback;)I
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
          }
      .end annotation
  .end method
