*** ./smali/android/view/ViewRootImpl$ImeInputStage.smali	2015-10-17 10:51:51.849421138 -0400
--- /home/jaskaran/Android/Patchrom6/nexus4/temp/new_smali/framework.jar.out/./smali/android/view/ViewRootImpl$ImeInputStage.smali	2015-10-17 10:46:44.812767735 -0400
***************
*** 104,109 ****
--- 104,120 ----
      move-result v2
  
      .local v2, "result":I
+      const/16 v3, 0x63
+ 
+      if-ne v2, v3, :cond_miui_0
+ 
+      const/4 v3, 0x0
+ 
+      return v3
+ 
+     :cond_miui_0
+     const/4 v3, 0x1
+ 
      if-ne v2, v3, :cond_0
  
      .end local v0    # "event":Landroid/view/InputEvent;
