*** ./smali/com/android/internal/telephony/Phone.smali	2015-10-17 10:52:20.876086550 -0400
--- /home/jaskaran/Android/Patchrom6/nexus4/temp/new_smali/telephony-common.jar.out/./smali/com/android/internal/telephony/Phone.smali	2015-10-17 10:47:07.092766772 -0400
***************
*** 376,381 ****
--- 376,384 ----
  .method public abstract getMessageWaitingIndicator()Z
  .end method
  
+ .method public abstract getMiuiIccPhoneBookInterfaceManager()Lcom/android/internal/telephony/MiuiIccPhoneBookInterfaceManager;
+ .end method
+ 
  .method public abstract getMsisdn()Ljava/lang/String;
  .end method
  
