*** ./smali/android/app/LoadedApk.smali	2015-10-17 10:50:59.216090078 -0400
--- /home/jaskaran/Android/Patchrom6/nexus4/temp/new_smali/framework.jar.out/./smali/android/app/LoadedApk.smali	2015-10-17 10:46:08.012769324 -0400
***************
*** 1812,1817 ****
--- 1812,1823 ----
  
      iput-object v0, p0, Landroid/app/LoadedApk;->mResources:Landroid/content/res/Resources;
  
+     iget-object v0, p0, Landroid/app/LoadedApk;->mResources:Landroid/content/res/Resources;
+ 
+     iget-object v1, p0, Landroid/app/LoadedApk;->mPackageName:Ljava/lang/String;
+ 
+     invoke-static {v0, v1}, Landroid/miui/ResourcesManager;->initMiuiResource(Landroid/content/res/Resources;Ljava/lang/String;)V
+ 
      :cond_0
      iget-object v0, p0, Landroid/app/LoadedApk;->mResources:Landroid/content/res/Resources;
  
