*** ./smali/com/android/server/net/NetworkPolicyManagerService.smali	2015-07-25 14:08:41.440876897 +0530
--- /home/jaskaran/Android/Patchrom/nexus4/temp/new_smali/services.jar.out/./smali/com/android/server/net/NetworkPolicyManagerService.smali	2015-07-25 14:05:43.184216051 +0530
***************
*** 4324,4329 ****
--- 4324,4331 ----
      iget-object v0, p0, Lcom/android/server/net/NetworkPolicyManagerService;->mActiveNotifs:Ljava/util/HashSet;
  
      invoke-virtual {v0}, Ljava/util/HashSet;->clear()V
+     
+     goto :cond_3
  
      invoke-direct {p0}, Lcom/android/server/net/NetworkPolicyManagerService;->currentTimeMillis()J
  
