*** ./smali/android/net/CaptivePortalTracker.smali	2015-07-11 20:07:20.909967026 +0530
--- /home/jaskaran/Android/Patchrom/nexus4/temp/new_smali/framework.jar.out/./smali/android/net/CaptivePortalTracker.smali	2015-07-11 20:05:23.586636194 +0530
***************
*** 730,735 ****
--- 730,745 ----
  
      iput-object v1, p0, Landroid/net/CaptivePortalTracker;->mUrl:Ljava/lang/String;
  
+     iget-object v1, p0, Landroid/net/CaptivePortalTracker;->mContext:Landroid/content/Context;
+ 
+     iget-object v2, p0, Landroid/net/CaptivePortalTracker;->mUrl:Ljava/lang/String;
+ 
+     invoke-static {v1, v2}, Landroid/net/CaptivePortalTrackerInjector;->getCaptivePortalServer(Landroid/content/Context;Ljava/lang/String;)Ljava/lang/String;
+ 
+     move-result-object v1
+ 
+     iput-object v1, p0, Landroid/net/CaptivePortalTracker;->mUrl:Ljava/lang/String;
+ 
      new-instance v1, Ljava/lang/StringBuilder;
  
      invoke-direct {v1}, Ljava/lang/StringBuilder;-><init>()V
