*** ./smali/android/net/IConnectivityManager.smali	2015-10-17 10:50:49.619423826 -0400
--- /home/jaskaran/Android/Patchrom6/nexus4/temp/new_smali/framework.jar.out/./smali/android/net/IConnectivityManager.smali	2015-10-17 10:45:53.792769939 -0400
***************
*** 447,456 ****
--- 447,472 ----
      .end annotation
  .end method
  
+ .method public abstract startUsingNetworkFeatureMSim(ILjava/lang/String;Landroid/os/IBinder;I)I
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract stopUsingNetworkFeature(ILjava/lang/String;)I
      .annotation system Ldalvik/annotation/Throws;
          value = {
              Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
+ .method public abstract stopUsingNetworkFeatureMSim(ILjava/lang/String;I)I
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
          }
      .end annotation
  .end method
