*** ./smali/com/android/server/BackupManagerService$PerformFullBackupTask.smali	2015-10-17 10:50:35.422757773 -0400
--- /home/jaskaran/Android/Patchrom6/nexus4/temp/new_smali/services.jar.out/./smali/com/android/server/BackupManagerService$PerformFullBackupTask.smali	2015-10-17 10:45:40.166103860 -0400
***************
*** 1411,1416 ****
--- 1411,1418 ----
  
      invoke-virtual {v5}, Ljava/io/FileOutputStream;->close()V
  
+     invoke-static {p2}, Lcom/android/server/BackupManagerService$Injector;->resetManifestFileModifiedTime(Ljava/io/File;)V
+ 
      return-void
  
      .end local v5    # "outstream":Ljava/io/FileOutputStream;
***************
*** 1726,1732 ****
  
      and-int v23, v23, v24
  
!     if-eqz v23, :cond_3
  
      move-object/from16 v0, v21
  
--- 1728,1734 ----
  
      and-int v23, v23, v24
  
!     #if-eqz v23, :cond_3
  
      move-object/from16 v0, v21
  
