*** ./smali/com/android/server/NotificationManagerService$5.smali	2015-10-17 10:50:23.356091627 -0400
--- /home/jaskaran/Android/Patchrom6/nexus4/temp/new_smali/services.jar.out/./smali/com/android/server/NotificationManagerService$5.smali	2015-10-17 10:45:31.466104236 -0400
***************
*** 382,387 ****
--- 382,394 ----
      # setter for: Lcom/android/server/NotificationManagerService;->mScreenOn:Z
      invoke-static {v1, v3}, Lcom/android/server/NotificationManagerService;->access$1502(Lcom/android/server/NotificationManagerService;Z)Z
  
+     move-object/from16 v0, p0
+ 
+     iget-object v1, v0, Lcom/android/server/NotificationManagerService$5;->this$0:Lcom/android/server/NotificationManagerService;
+ 
+     # invokes: Lcom/android/server/NotificationManagerService;->updateNotificationPulse()V
+     invoke-static {v1}, Lcom/android/server/NotificationManagerService;->access_updateNotificationPulse(Lcom/android/server/NotificationManagerService;)V
+ 
      goto :goto_4
  
      :cond_f
