*** ./smali/android/net/wifi/WifiConfiguration$KeyMgmt.smali	2015-10-17 10:50:47.792757239 -0400
--- /home/jaskaran/Android/Patchrom6/nexus4/temp/new_smali/framework.jar.out/./smali/android/net/wifi/WifiConfiguration$KeyMgmt.smali	2015-10-17 10:45:52.306103336 -0400
***************
*** 35,41 ****
      .locals 3
  
      .prologue
!     const/4 v0, 0x5
  
      new-array v0, v0, [Ljava/lang/String;
  
--- 35,41 ----
      .locals 3
  
      .prologue
!     const/4 v0, 0x7
  
      new-array v0, v0, [Ljava/lang/String;
  
***************
*** 69,74 ****
--- 69,86 ----
  
      aput-object v2, v0, v1
  
+     const/4 v1, 0x5
+ 
+     const-string v2, "WAPI_PSK"
+ 
+     aput-object v2, v0, v1
+ 
+     const/4 v1, 0x6
+ 
+     const-string v2, "WAPI_CERT"
+ 
+     aput-object v2, v0, v1
+ 
      sput-object v0, Landroid/net/wifi/WifiConfiguration$KeyMgmt;->strings:[Ljava/lang/String;
  
      return-void
