*** ./smali/android/net/wifi/WifiEnterpriseConfig$Eap.smali	2015-10-17 10:50:49.039423851 -0400
--- /home/jaskaran/Android/Patchrom6/nexus4/temp/new_smali/framework.jar.out/./smali/android/net/wifi/WifiEnterpriseConfig$Eap.smali	2015-10-17 10:45:53.342769957 -0400
***************
*** 33,39 ****
      .locals 3
  
      .prologue
!     const/4 v0, 0x4
  
      new-array v0, v0, [Ljava/lang/String;
  
--- 33,39 ----
      .locals 3
  
      .prologue
!     const/4 v0, 0x6
  
      new-array v0, v0, [Ljava/lang/String;
  
***************
*** 61,66 ****
--- 61,78 ----
  
      aput-object v2, v0, v1
  
+     const/4 v1, 0x4
+ 
+     const-string v2, "SIM"
+ 
+     aput-object v2, v0, v1
+ 
+     const/4 v1, 0x5
+ 
+     const-string v2, "AKA"
+ 
+     aput-object v2, v0, v1
+ 
      sput-object v0, Landroid/net/wifi/WifiEnterpriseConfig$Eap;->strings:[Ljava/lang/String;
  
      return-void
